couchbase-on-ubuntu/azuredeploy.json

```diff
@@ -69,27 +69,27 @@
     },
     "cbPackageDownloadBase": {
       "type": "string",
-      "defaultValue": "http://packages.couchbase.com/releases/3.0.3/",
+      "defaultValue": "http://packages.couchbase.com/releases/4.1.1/",
       "metadata": {
         "description": "Couchbase package download location"
       }
     },
     "cbPackage": {
       "type": "string",
-      "defaultValue": "couchbase-server-enterprise_3.0.3-ubuntu12.04_amd64.deb",
+      "defaultValue": "couchbase-server-enterprise_4.1.1-ubuntu12.04_amd64.deb",
```

Comment:
it is good to expose both to help users override. some of these URLs have changed in the past.

for the second part, I am happy to extend the template. Ubuntu is one of the top platforms so this should cover a good chunk of new trials. if you have a template I can cheat off of, pls send that my way and I'll see if I can add platform agnostic option in.